USAGE.adoc

```diff
@@ -50,3 +50,22 @@ Writing output/deprecated-and-unused.html
 Writing output/usage-by-api.json
 Writing output/usage-by-api.html
 ----
+
+=== Finding Usage of Methods and Fields
+
+The command line flag `-M filename.txt` or `--additionalMethods filename.txt` allows specific methods to be searched for.
+The additional flag `-i` or `--onlyIncludeSpecified` will filter the output report to only show usage of the methods specified.
+For example, to find usage of `hudson.util.Secret.toString`, the following line should be included in the specified file:
+
+[source]
+----
+hudson/util/Secret#toString
+----
+
+Class names should be specified in their fully qualified form (i.e., inner classes should be referenced via `Outer$Inner` rather than by their canonical names).
```

Comment:
More nit:
```suggestion
Specify class names in their fully qualified form (i.e., inner classes should be referenced via `Outer$Inner` rather than by their canonical names).
```